pkgs/applications/misc/octoprint/printtimegenius-logging.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/setup.py b/setup.py
+index 6a6610e..cc45902 100644
+--- a/setup.py
++++ b/setup.py
+@@ -35,9 +35,9 @@ plugin_license = "AGPLv3"
+ # Any additional requirements besides OctoPrint should be listed here
+ # For now, require the working release, which is only 1.3.9rc1.
+ plugin_requires = ["OctoPrint>=1.3.9rc1", "psutil", "sarge"]
+-from sys import version_info
+-if version_info[0] < 3:
+-    plugin_requires.append("logging")
```

Comment:
Why do we need to disable the `logging` package?

Reply:
They added these lines because of this issue:
https://github.com/eyal0/OctoPrint-PrintTimeGenius/issues/176

However, in our environment, "logging" isn't a package and can't be found, so it breaks the build. (It's in the Python standard library.) I don't think commenting this out actually disables logging, I think it just removes its requirement as a plugin.

Reply:
Ok, that makes sense.